base/common/src/org/dogtagpki/tps/main/Util.java

```diff
@@ -790,329 +566,4 @@ public static String getTimeStampString(boolean addMicroSeconds) {
 
         return timeString;
     }
-
-    //AES CMAC test samples
-    public static void main(String[] args) {
```

Comment:
We could do taht, but I figured the tests you have in KBDF covers this ground???? I wanted to just have it there for viewing / informational purpose, but feel free to do whatever you need, if you think it could help somewhere.

Reply:
@jmagne I have extensive tests against NIST's code in NSS, and three "sanity" checks in JSS. I don't have any that checked that what NIST expect and what SCP03 expect align, which is what this'd validate. :-)